apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V25__Add_partd_contract_number_index.sql

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Sorry, one last thing: I'd add a header comment here that explains the reason for the change and references the JIRA ticket.

I wouldn't do that everywhere, but makes sense for DB migrations to me, as I often read them start - end, and I suspect other folks do, too. Good chance to help folks understand how & why the app has evolved over time.

Reply:
good call, done.